### ISSUE TITLE
Problem with eups expandbuild

### ISSUE THREAD

**Issue by garmilla** [RobertLuptonTheGood/eups#71]
I ran into another problem. This time I got the following error when running `eups expandbuild tableFile`:
```
eups expandbuild: global name 'distrib' is not defined
```

Applying the following change to my installed eups fixed the problem:
```
diff --git a/python/eups/app.py b/python/eups/app.py
index 33b9afe..3e67397 100644
--- a/python/eups/app.py
+++ b/python/eups/app.py
@@ -404,7 +404,7 @@ def expandBuildFile(ofd, ifd, product, version, svnroot=None, cvsroot=None, repo
     if svnroot:
         builderVars["SVNROOT"] = svnroot
 
-    distrib.builder.expandBuildFile(ofd, ifd, product, version, verbose, builderVars,
+    builder.expandBuildFile(ofd, ifd, product, version, verbose, builderVars,
                                     repoVersionName=repoVersion)
```

**Comment by timj** [comment]
This will be fixed when #69 is merged.

**Comment by garmilla** [comment]
Thanks Tim!

**Comment by mjuric** [comment]
Merged #69 & considering this one fixed as well; if the issue persists, plz. reopen!
